src/behavior-router.ts

```diff
@@ -55,13 +56,32 @@ export class BehaviorRouter {
 		this.cfResources = serverless.service?.resources?.Resources || {};
 		this.cacheDir = path.resolve(options.cacheDir || path.join(os.tmpdir(), 'edge-lambda'));
 		this.fileDir = path.resolve(options.fileDir || path.join(os.tmpdir(), 'edge-lambda'));
-		this.path = this.serverless.service.custom.offlineEdgeLambda.path || '';
+		this.path = this.serverless.service.custom.offlineEdgeLambda.path || '.';
```

Comment:
I believe changing the default from '' to '.' should have no effect. However, it has not been explicitly tested.

Reply:
Opted to not change the default and use path.join instead.